src/db_window.rs

```diff
@@ -19,7 +19,7 @@ use std::sync::atomic::AtomicUsize;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::{Arc, RwLock};
 
-pub const MAX_REPAIR_LENGTH: usize = 128;
+pub const MAX_REPAIR_LENGTH: usize = 8192;
```

Comment:
do we expect this to make a difference?

Reply:
I see in the new graph that it's picking up more indices for the repair request. Without this, the repair requests were staying close to consume, while the gap between receive and consume was rapidly increasing.

Reply:
Okay. We should run this by @carllin as well.

Reply:
Sure, added @carllin to the review

Reply:
I'd bust this hunk out into a separate PR so we don't block on getting the rest of this PR landed.  r+ on the rest and it can land anytime